google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,248 @@
+  python:
+    package_name: google.cloud.iam.admin.v1
+  go:
+    package_name: cloud.google.com/go/iam/admin/apiv1
+  csharp:
+    package_name: Google.Iam.Admin.V1
+  ruby:
+    package_name: Google::Cloud::Iam::Admin::V1
+  php:
+    package_name: Google\Cloud\Iam\Admin\V1
+  nodejs:
+    package_name: '@google-cloud/admin'
+interfaces:
+- name: google.iam.admin.v1.IAM
+  collections:
+  - name_pattern: projects/{project}
+    entity_name: project
+  - name_pattern: projects/{project}/serviceAccounts/{service_account}
+    entity_name: service_account
+  - name_pattern: projects/{project}/serviceAccounts/{service_account}/keys/{key}
+    entity_name: key
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 20000
+    total_timeout_millis: 600000
+  methods:
+  - name: ListServiceAccounts
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: true
+    page_streaming:
+      request:
+        page_size_field: page_size
+        token_field: page_token
+      response:
+        token_field: next_page_token
+        resources_field: accounts
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: project
+    timeout_millis: 60000
+  - name: GetServiceAccount
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: service_account
+    timeout_millis: 60000
+  - name: CreateServiceAccount
+    flattening:
+      groups:
+      - parameters:
+        - name
+        - account_id
+        - service_account
+    required_fields:
+    - name
+    - account_id
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: project
+    timeout_millis: 60000
+  - name: UpdateServiceAccount
+    flattening:
+      groups:
+      - parameters:
+        - name
+        - project_id
+        - unique_id
+        - email
+        - display_name
+        - etag
+        - oauth2_client_id
+    required_fields:
+      - etag
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: service_account
+    timeout_millis: 60000
+  - name: DeleteServiceAccount
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: false
+    retry_codes_name: idempotent
```

Comment:
Similarly the [http request](https://github.com/googleapis/googleapis/pull/151/files/1654ad516f42672c895995cd50a1203fd3ad2dc3#diff-dba0a0fe59808a0ccad679f0547189e5R56) is get thus idempotence.